client/lib/abtest/active-tests.js

```diff
@@ -191,6 +191,14 @@ export default {
 			variantUXImprovements: 50,
 		},
 		defaultVariation: 'control',
+	},
+	showCheckoutCartRight: {
+		datestamp: '20190502',
+		variations: {
+			right: 50,
+			original: 50,
```

Comment:
Let's try to stick to a naming convention since I've found it can make more straightforward later understanding results, post-mortem, etc. 

I realize this wasn't documented anywhere (not sure what would be the best place for it though!), but I'm always sticking to have `control` and `variant` prepended (for easier quick identification) and then followed by some small description (usually not needed for `control`). 

In this case, I'd rename this to:

- `variantRightColumn`
- `controlLeftColumn` or just `control`

Reply:
Done in [8fccb3b](https://github.com/Automattic/wp-calypso/pull/32832/commits/8fccb3bd402fe58a20a785c4c804f5a608584ecf)